### ISSUE TITLE
#Importing the Libraries import numpy as np import pandas as pd import datetime import matplotlib import matplotlib.pyplot as plt from matplotlib import colors import seaborn as sns from sklearn.preprocessing import LabelEncoder from sklearn.preprocessing import StandardScaler from sklearn.decomposition import PCA from yellowbrick.cluster import KElbowVisualizer from sklearn.cluster import KMeans import matplotlib.pyplot as plt, numpy as np from mpl_toolkits.mplot3d import Axes3D from sklearn.cluster import AgglomerativeClustering from matplotlib.colors import ListedColormap import plotly.graph_objects as go from plotly.subplots import make_subplots from plotly.offline import init_notebook_mode, iplot init_notebook_mode(connected=True) from sklearn import metrics import warnings import sys if not sys.warnoptions:     warnings.simplefilter("ignore") np.random.seed(42) from pandas_profiling import ProfileReport

### ISSUE THREAD

**Issue by thaofee2410** [thaofee2410/Thao#1]
abc
